springfox-staticdocs/build.gradle

```diff
@@ -14,6 +14,21 @@
  * limitations under the License.
  */
 
+configurations {
+    testArtifacts.extendsFrom testRuntime
+}
+task testJar(type: Jar) {
+    classifier "test"
+    from sourceSets.test.output
+    exclude '**/*log4j.xml*', '**/*simplelogger.properties*'
+}
+artifacts {
+    testArtifacts testJar
+}
+
 dependencies {
-  testCompile libs.test
+    compile "io.github.robwin:swagger2markup:0.3.0"
+    compile libs.test
```

Comment:
No, because I need spring-test at Compiler time.
This was just the first Version so that I dont have to modify dependencies.gradle.